pkg/config/nodetreemodel/read_config_file.go

```diff
@@ -19,7 +20,12 @@ import (
 
 func (c *ntmConfig) getConfigFile() string {
 	if c.configFile == "" {
-		return "datadog.yaml"
+		for _, path := range c.configPaths {
+			configFilePath := filepath.Join(path, c.configName+".yaml")
```

Comment:
What if `c.configName` is not set? Should it default to `"datadog"` in that case?

Reply:
Good catch, I think initializing `configName` in `NewConfig` is better (I updated the PR).